### ISSUE TITLE
[Feature request] Add Estonian road maps from Estonian National Geoportal

### ISSUE THREAD

**Issue by macintoshee** [osmandapp/Osmand#7702]
In most countries OSMAND is better than any other map service. Due the very low population density and low overall population this statement is not true regarding to Estonia. For example, one major crossroad near capital city was not updated in OSMAND for 3 years. There is just not enough editors. The same time Estonian National Geoportal has all the road related data updated constantly and its much better than current OpenStreet maps (or any other maps service). It covers all the roads and tracks known to Estonian Land Board (and they do know a lot). Usage of Estonian National Geoportal data is open and free. Only requirement is referring to a source ie to Estonian Land Board. 

http://inspire.maaamet.ee/home
http://inspire.maaamet.ee/services/transport-networks-(tn)
https://geoportaal.maaamet.ee/eng/Services/Public-WMS-service-p346.html

**Comment by vshcherb** [comment]
We're not planning to use any other data source that is competing with OpenStreetMap, so people will fell more encouraged to contribute to OSM

**Comment by SviMik** [comment]
Why didn't you fix it yourself if it was bugging you for 3 years?
If it's still not there - please send me the link, I'll look into this.

OSM is not that bad in Estonia. Have you noticed, for example, that it has almost all addresses? Or that many buildings, even in remote areas, are present too?

**Comment by macintoshee** [comment]
You misunderstood me, I did fix it as soon as I discovered it. But it was three years since new road, a major one, was built and entire crossroad was rebuilt to multilevel. Someone just pointed out this as example how outdated Open Street Maps in Estonia are. And I had to agree with him.
I do take a part rogaining (kind of orienteering) events and OSMAND is preferred for navigation there (there is really no good alternative for that purpose). During that I see lots of small and strange roads not represented in OSMAND. Very often those are not new roads but rather existed long time but are not travelled that often and therefore not represented here. 
Densely populated areas are not a problem but rural areas are quite often not satisfactory.
